pkg/agent/metrics/prometheus.go

```diff
@@ -94,6 +94,30 @@ var (
 		},
 		[]string{"operation"},
 	)
+
+	TotalConnectionsInConnTrackTable = metrics.NewGauge(
+		&metrics.GaugeOpts{
+			Name:           "antrea_agent_conntrack_total_connection_count",
+			Help:           "Number of connections in the conntrack table, update every flowPollInterval.",
+			StabilityLevel: metrics.ALPHA,
+		},
+	)
+
+	TotalAntreaConnectionsInConnTrackTable = metrics.NewGauge(
+		&metrics.GaugeOpts{
+			Name:           "antrea_agent_conntrack_antrea_connection_count",
+			Help:           "Number of connections in the Antrea ZoneID of the conntrack table.",
```

Comment:
Above flowPollInterval statement is applicable here too.

Reply:
Antrea connection count is updated when add or delete a connection, so I don't think the flowPollInterval statement applies here.

Reply:
Add and delete connection logic is only executed through connectionStore.Poll which is called at a given flowPollInterval, so the metric update is dependent on it.

Reply:
Got it, thanks!